Userland/Libraries/LibAudio/Sample.h

```diff
@@ -77,6 +79,18 @@ struct Sample {
         return log(val / VOLUME_A) / VOLUME_B;
     }
 
+    // Decibels are a logarithmic "unit":
+    // 20 dB = 10x difference between two loudness levels.
+    static ALWAYS_INLINE float db_to_linear(float const val)
+    {
+        return pow(10.0f, val / 20.0f);
```

Comment:
```suggestion
        return pow(10.0f, val * 0.05f);
```

Reply:
Is this more precise? The division by 20 is much more intuitive especially if you don't know how the Decibel scale works.

Reply:
Well it is a division in a possibly hot code path,
Accuracy should be the same for our ears

Also decibel is weird, in some cases, like with pure energy, it has multiplier ten, but in a lot of actual use cases it has a 20 multiplier, because somewhere down the line there's a sqrt in the formulas....

Tldr: I'd say your choice, mult is faster division is more clear, in doubt comments are always an option

Reply:
Actually, factor 10 is only used for dBV, i.e. voltage decibels. But we're not electrical engineers :^)